src/ngraph_builder.cc

```diff
@@ -41,6 +41,15 @@ const static std::map<tf::DataType, ngraph::element::Type> TF_NGRAPH_TYPE_MAP =
      {tf::DataType::DT_UINT16, ng::element::u16},
      {tf::DataType::DT_BOOL, ng::element::boolean}};
 
+static tf::Status ValidateInputCount(const tf::Node* op, size_t count) {
+  if (op->num_inputs() != count) {
+    return tf::errors::InvalidArgument(
+        op->name(), " requires ", count, " input(s), got ", op->num_inputs(),
```

Comment:
This is very good error message - descriptive - so that users know what to do. Only suggestion would be to provide \" around the op name.